template/.claude/commands/spike.md

```
@@ -0,0 +1,103 @@
+---
+description: Execute TDD Spike Phase - exploratory coding to understand problem space before TDD
+argument-hint: <exploration description>
+---
+
+**User arguments:**
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)